out_writeapi.go

```diff
@@ -262,6 +263,12 @@ func FLBPluginInit(plugin unsafe.Pointer) int {
 	datasetID := output.FLBPluginConfigKey(plugin, "DatasetID")
 	tableID := output.FLBPluginConfigKey(plugin, "TableID")
 
+	//set exactly-once bool from config file param
+	exactlyOnceVal, err := strconv.ParseBool(output.FLBPluginConfigKey(plugin, "Exactly_Once"))
```

Comment:
If the parameter is absent we should default to False.

Reply:
Done.